doc/apps/s_server.pod

```diff
@@ -144,6 +145,11 @@ a static set of parameters hard coded into the s_server program will be used.
 if this option is set then no DH parameters will be loaded effectively
 disabling the ephemeral DH cipher suites.
 
+=item B<-no_ecdhe>
+
+if this option is set then no ECDH curves will be selected, effectively
```

Comment:
I agree with @InfoHunter that this entry should replace curves with parameters, for coherence with the equivalent `-no_dhe` option.

Reply:
technically, the only parameter the server can select in ECDH *is* the curve – the base point is bound to the curve so is not selectable – in FFDH it's both the prime and the generator that is selectable and sent to the client so there are parameter*s* (plural) in SKE

so for ECDH "curves" and "parameters" in this context are synonymous